src/display/api.js

```diff
@@ -281,8 +283,18 @@ function getDocument(src, pdfDataRangeTransport,
       if (task.destroyed) {
         throw new Error('Loading aborted');
       }
+      let data = { source: params,
+                   disableRange: getDefaultSetting('disableRange'), };
```

Comment:
we don't need to create data temporate, let's use `params` directly with PDFDataTransportStream. For PDFNetworkStream inline the object literal as parameter.